### DISCUSSION TITLE
SAW Filter for 162MHz - where to buy ???

### DISCUSSION THREAD

**Discussion by OE1MWW** [jvde-github/AIS-catcher#198] [General]
In my spectrum view I noticed that there was a pager transmitter in my area on 162.475 Mhz. Unfortunately, the carrier is only activated to transmit messages, so the interference is very irregular. In the discussion here, a SAW filter from TAI-SAW TECHNOLOGY CO., LTD. Type TB0436A was mentioned. https://www.taisaw.com/upload/product/TB0436A%20_Rev.1.0_.pdf
The search on the net was very difficult, DigiKey demands 5 USD, but stock level 'zero'. I then sent an inquiry to several semiconductor distributors (China / Hong Kong) - result: horror price - up to 50 USD per piece + 36 USD handling fee and 30 USD shipping.
 But I found what I was looking for at Semtke Electronic Company Limited  http://www.semtke.com/ The TB0436A for 7.75 USD / pc + 9 USD shipping to Europe, PayPal payment.  Tip: send an email to "iven xie <abbie@semtke.com>" right away and ask for a quote.
My filters should be in my mailbox in about 2 weeks from now.

greetings from Vienna Austria
OE1MWW

**Comment by pbriesch** [comment]
As far as I know the rtl-sdr dongle fom ShipXplores already a build in filter: https://www.amazon.de/-/en/ShipXplorer-AIS-Dongle-Blue/dp/B09Y61CM12

And this might be interesting for you as well, a filtered pre-amplifier for AIS: https://store.uputronics.com/index.php?route=product/product&path=59&product_id=93

73, DD8PL

**Reply by jvde-github** [reply]
Agree. I need a filter as well and the ShipXplorer dongle solved it for me (has indeed a filter) but also the RTL-SDR V3 in combination with the GPIO Labs AIS passband works well. The shipping cost makes the GPIO Labs a relatively expensive option.

**Reply by pbriesch** [reply]
Here a link with some information about the ShipXplorer dongel and the Uputronics preamplifier:

[https://discussions.flightaware.com/t/howto-run-ads-b-receiver-ais-receiver-on-same-rpi/85040/85?page=5](https://discussions.flightaware.com/t/howto-run-ads-b-receiver-ais-receiver-on-same-rpi/85040/85?page=5)

**Reply by Taxom** [reply]
And it is in this topic that I explain why this SAW-filter is useless for our case.  It has bandwidth from 156.3 to 162.025 (all marine VHF band) and we need only 80kHz bandwidth (161.975MHz-12.5kHz to 162.025MHz+12.5kHz)

**Reply by jvde-github** [reply]
Out of curiosity, have you seen any benefits of using the RTL filter via -a? For me it does not make a difference but heard from others it can be a big improvement...

**Reply by Taxom** [reply]
If I right understand correctly, the parameter -a sets a built in hardware tunable IF-filter. It’s hard for me to say, I installed this parameter from the very beginning when I had no SAW-filter. And now this parameter is most likely not to influence my installation. He can likely filter the neighboring radio channels, but only if they do not overload (saturation) the receiver's LNA, and most likely it happened in my case due to weather broadcasting (more then +30db above the most powerful AIS messages). 
 
![820](https://github.com/jvde-github/AIS-catcher/assets/20313019/03676461-1f26-471d-8418-1b655e84091e)

**Reply by Taxom** [reply]
There is why to important has input narrow bandpass filter if u have a power broadcast near.
The SDR has very poor adjacent-channel selectivity and too small dynamic range for In order to select weak signals against the background of strong interference.
https://www.youtube.com/watch?v=btDiWHuI730

**Reply by hdtvspace** [reply]
The airnav ADS-B flightsticks and AIS shipxplorer USB sticks have an cheap filter before the LNA at the input signal. This means an high insertion signal loss. No metal case, no TCO and so on…  These dongles are only for the easy entry / beginner.

**Comment by OE1MWW** [comment]
An TA0395C filter is installed in the ShipXplorer. This filter is wider than the TB0436A I ordered.
And my 'interferer', the pager transmitter on 162.475 Mhz, would still fall well within the filter curve
of the TA0395C in the ShipXplorer. Have to built 'my own' filter ;-)

Am Mo., 20. Nov. 2023 um 17:28 Uhr schrieb Jasper ***@***.***

**Comment by Phil353556** [comment]
Hi @OE1MWW  @pbriesch 
Do you have some feedbacks to share with the use of this pre-amplifier for AIS?
Thank you

**Reply by OE1MWW** [reply]
@Phil353556

The TB0436A is a passive filter, not an pre-amplifier.
The mentioned nearby transmitter is a pager system. The carrier comes and goes as
messages are sent. Every time there is this signal, the AGC of my RX starts to act.
My MSI.SDR receiver has a very wide band input stage. With or without that filter
in front, there is not much differency in signal level of the received AIS - checked 
again, the decoding is the same. Insertion loss of the filter is ~ -3db. And yes, the
pager signal has less impact by using the filter.

The picture shows my home made board according to the circuit diagram on the 
datasheet of the TB0436A. (Google 'TB0436A', there are many links). Prices for the
filter vary a lot, depending on the dealer and the availability you may find.

![saw](https://github.com/jvde-github/AIS-catcher/assets/73303392/5bce8f3b-3f4b-41a6-8610-64b4f90cd671)

**Reply by Taxom** [reply]
Just yesterday I made another one for off grid use.
 
![photo_2024-04-29_08-38-06](https://github.com/jvde-github/AIS-catcher/assets/20313019/8dde9a42-1a9d-4981-9b52-462d86067551)
![photo_2024-04-29_08-38-10](https://github.com/jvde-github/AIS-catcher/assets/20313019/9fb30626-52cd-40a0-a953-64f4351e7974)
![photo_2024-04-29_08-38-08](https://github.com/jvde-github/AIS-catcher/assets/20313019/99275086-5195-49b1-9af1-2cfe5bb11473)

my first one here https://github.com/jvde-github/AIS-catcher/discussions/194

**Reply by OE1MWW** [reply]
interresting, mine looks a little bit different ;-)
![AIS SAW Filter nanovna-h_2024 04 29-15 42](https://github.com/jvde-github/AIS-catcher/assets/73303392/0fcea368-d803-4002-ad25-42e4a5bb8c85)

**Reply by Taxom** [reply]
I double checked with high resolution (1000p/1MHz)

![photo_2024-04-29_11-13-40](https://github.com/jvde-github/AIS-catcher/assets/20313019/d8556e94-b5f6-4658-b97e-cfcbe510b929)
![162F_500W_HD](https://github.com/jvde-github/AIS-catcher/assets/20313019/de593972-df7d-4dc4-8aae-0a51d4a6ff16)
